src/core/Akka.Remote/Endpoint.cs

```diff
@@ -1500,7 +1500,7 @@ private bool WriteSend(EndpointManager.Send send)
             {
                 _log.Error(
                   ex,
-                  "Serializer not defined for message type [{0}]. Transient association error (association remains live)",
+                  "Serialization failed for message [{0}]. Transient association error (association remains live)",
```

Comment:
Made this message less specific, since we don't necessarily know why an outbound write failed.

Reply:
(the inner exception should tell us)